pyproject.toml

```diff
@@ -5,7 +5,7 @@ requires = [
 build-backend = "flit_core.buildapi"
 
 [project]
-name = "ansys-pyensight-core"
+name = "ansys-ensight-core"
```

Comment:
@jorgepiloto it was decided to name the project ansys-pyensight-core and not ansys-ensight-core since the module is shipped with EnSight and in the EnSight Python APIs there's already an ensight module.
It is vital for us that this happens, and so to avoid conflicts we kept the name pyensight, in agreement with @RobPasMue  and @AlejandroFernandezLuces

Reply:
Thanks for pointing this out. I thought it was a mistake. All good then.